virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,18 +1,26 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to an external network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+{VirtProductName} is installed with a default internal `pod` network. You update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration] depending on the type of network to which you want to connect.
+
+[IMPORTANT]
+====
+When connecting to an external network, it is essential to first create a _Linux bridge_. Until you create a Linux bridge, devices such as network interface card (NICs) are not available because your external network is not yet exposed. To expose your external network and make such devices available, you first need to create a Linux bridge.
+====
+
+Once a network bridge is created, you can start virtual machines on the external network in the same manner that you start virtual machines attached to internal networks, by using PXE booting, for example.
```

Comment:
You can remove lines 16 and 18. PXE booting, by definition, requires an external network. There are thousands of reasons why a user needs external networks (HTTP servers, backup/restore, disaster recovery, CI/CD deployments). We can't possibly list them all, so we should probably not even try. That's why I think we should remove this line.

Reply:
Deleted.